CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# v1.10.1 (2022-10-19)
+
+## OS Changes
+* Support container runtime settings: enable-unprivileged-icmp, enable-unprivileged-ports, max-concurrent-downloads, max-container-log-line-size ([#2494])
+* Update EKS-D to 1.22-11 ([#2490])
+* Update EKS-D to 1.23-6 ([#2488])
```

Comment:
Small nit:
I worry that people might mistake `1.22-11` as the K8s version when it's the EKS-D release version. 
EKS-D `1.22-11` maps to K8s `v1.22.15`. But maybe the `-` makes it clear?